### PR TITLE
Optimization November 22 - Examples

### PR DESCRIPTION
**Pull request by adil-aspose**

### CODE REVIEW

File: english/net/aspose.slides/presentation/customdata/_index.md [hunk 1]
```diff
@@ -14,6 +14,19 @@ Returns the presentation's custom data. Read-only [`ICustomData`](../../icustomd
 public ICustomData CustomData { get; }
 ```
 
+## Examples
+
+The following example shows how to add a tag to a PowerPoint Presentation.
```
Comment (yury-knml) [inline english/net/aspose.slides/presentation/customdata/_index.md:19]: Does this example seem to be more appropriate for TagCollection, not CustomData?

**Comment by adil-aspose** [comment]
Thanks for feedback. Reported issues have been resolved. Please review.

**Comment by stasmiroshnichenko** [comment]
@adil-aspose Please do not merge these changes into the main branch, as this will cause conflicts with our changes and will be overwritten anyway when automatically generating the API reference for the new version. We will make the necessary changes in the code so that your changes are not lost.

**Comment by adil-aspose** [comment]
@stasmiroshnichenko Sure, Please change source comments so change can be permanent. I have closed the pull request.

**Pull request closed without merging**